fixieai/cli/auth/user_config.py

```diff
@@ -0,0 +1,40 @@
+import dataclasses
+import os
+from typing import Optional
+
+from fixieai.cli import utils
+
+
+@dataclasses.dataclass
+class UserConfig(utils.DataClassYamlMixin):
+    """Represents user config at ~/.config/fixie/config.yaml"""
+
+    fixie_api_key: Optional[str] = None
```

Comment:
```suggestion
    fixie_access_token: Optional[str] = None
```

Reply:
Hmm api_key is the terminology we use on the platform (http://app.fixie.ai/profile) and the environment variable `FIXIE_API_KEY`. I think changing it right now will cause more confusion than clarity. The right next step is probably to make it be an api key instead of a full platform access token. Wdyt?